apps/test/unit/lib/kits/maker/PlaygroundComponentsTest.js

```diff
@@ -84,7 +90,8 @@ describe('Circuit Playground Components', () => {
           let led;
 
           beforeEach(() => {
-            led = createCircuitPlaygroundComponents(board).colorLeds[pin];
+            return createCircuitPlaygroundComponents(board)
+              .then(({colorLeds}) => led = colorLeds[pin]);
```

Comment:
You have two options: You can use the `done` method, or you can return a `Promise`.  We use the latter strategy here.